iulianIachimov/src/main/java/datacollectiontask/Models/Order.java

```diff
@@ -0,0 +1,39 @@
+package datacollectiontask.Models;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+public class Order {
+    public long id;
+    public LocalDateTime date;
+    public User user;
+    public List<Item> items;
+
+    static long idCounter;
+    // private Random rand = new Random(); // generating random ID
+
+    public Order(User user, List<Item> items) {
+        // this.id = rand.nextInt();
+        this.id = ++idCounter;
```

Comment:
better use postincrement

Reply:
When using post increment  it will assign '0' value to ID, that is why I have chosen pre increment. Let me know if I still need to change it by assigning an initial value to idCounter.